src/Symfony/Component/Validator/Constraints/DefaultPasswordStrengthEstimator.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Constraints;
+
+final class DefaultPasswordStrengthEstimator implements PasswordStrengthEstimatorInterface
```

Comment:
We can find a more descriptive term than "default" to name this class. Like "WordCount".

Reply:
What about `EntropyBasedPasswordStrengthEstimator`?